### PR TITLE
Specifies a hide keyword tag into Mute Lists

### PR DESCRIPTION
**Pull request by vitorpamplona**
Users can add keywords to their Mute list and Clients can then filter posts that contain those keywords out of the view.

### CODE REVIEW

**Comment by jb55** [comment]
could call it "word" since it's muting a word?

**Comment by vitorpamplona** [comment]
Wouldn't that be confusing? People can see that as following the word 🤔

**Comment by vitorpamplona** [comment]
On a second thought, the context of the tag is always decided by the list, so `word` it is.

**Review by staab — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -191,6 +191,7 @@ When experimenting with kinds, keep in mind the classification introduced by [NI
 | `summary`         | article summary                      | --                   | [23](23.md)              |
 | `thumb`           | badge thumbnail                      | dimensions in pixels | [58](58.md)              |
 | `title`           | article title                        | --                   | [23](23.md)              |
+| `word`            | words                                | --                   | [51](51.md)              |
```
Comment (mattn) [inline README.md:194]: Is this `wordP a generic tag? If not, should it be marked `must words`?

**Review by fiatjaf — APPROVED** [review]

**Comment by fiatjaf** [comment]
Time to merge this?

**Comment by mikedilger** [comment]
+1

**Comment by vitorpamplona** [comment]
Implemented on Amethyst [v0.78.0 Mute words](https://github.com/vitorpamplona/amethyst/releases/tag/v0.78.0)

**Comment by fiatjaf** [comment]
I believe this is fully included in https://github.com/nostr-protocol/nips/pull/880 now.

**Pull request closed without merging**